core/domain/feedback_services.py

```diff
@@ -172,38 +243,69 @@ def get_next_page_of_all_feedback_messages(
-    """Returns a dict with feedback thread analytics for the given exploration
-    ids.
+    """Fetches all FeedbackAnalytics, for all the given exploration ids.
+
+    A FeedbackAnalytics contains the exploration id the analytics
+    belongs to, how many open threads exist for the exploration,
+    how many total threads exist for the exploration.
+
+    Args:
+        exploration_ids: list of str. A list of exploration ids.
+
+    Returns:
+        list of FeedbackAnalytics. It's in the the same order as the input
+        list. If the exploration id is invalid, the number of threads will
+        be zero in the corresponding FeedbackAnalytics object.
     """
     return feedback_jobs_continuous.FeedbackAnalyticsAggregator.get_thread_analytics_multi( # pylint: disable=line-too-long
         exploration_ids)
 
 
 def get_thread_analytics(exploration_id):
-    """Returns a dict with feedback thread analytics for the given exploration.
+    """Fetches the FeedbackAnalytics for the given exploration id.
 
-    The returned dict has two keys:
-    - 'num_open_threads': the number of open feedback threads for this
-        exploration.
-    - 'num_total_threads': the total number of feedback threads for this
-        exploration.
+    Args:
+        exploration_id: str.
+
+    Returns:
+        list of FeedbackAnalytics.
     """
     return feedback_jobs_continuous.FeedbackAnalyticsAggregator.get_thread_analytics( # pylint: disable=line-too-long
         exploration_id)
 
 
 def get_total_open_threads(feedback_thread_analytics):
-    """Returns the count of all open threads for the given
-    FeedbackThreadAnalytics domain objects."""
+    """Gets the count of all open threads for the given FeedbackThreadAnalytics
+    domain object.
+
+    Args:
+        feedback_thread_analytics: FeedbackThreadAnalytics.
+
+    Returns:
+        int. The count of all open threads for the given FeedbackThreadAnalytics
+        domain object.
+    """
     return sum(
         feedback.num_open_threads for feedback in feedback_thread_analytics)
 
 
 def create_suggestion(exploration_id, author_id, exploration_version,
                       state_name, description, suggestion_content):
-    """Creates a new SuggestionModel object and the corresponding
-    FeedbackThreadModel object."""
+    """Creates a new SuggestionModel and the corresponding FeedbackThreadModel
+    domain object.
 
+    Args:
+        exploration_id: str. The exploration id the suggestion belongs to.
+        author_id: str. ID of the user who submitted the suggestion.
+        exploration_version: int. The exploration version for
+            which the suggestion was made.
+        state_name: str or None. The state name for the thread. If None,
+            this indicates that the thread pertains to the exploration as a
+            whole.
+        description: str. Learner-provided description of suggestion changes.
+        suggestion_content: dict. Multiple key-value pairs. Key is the
```

Comment:
From feedback_service_test.py, suggestion_content has been passed in twice.
```python
feedback_services.create_suggestion(
                self.EXP_ID2, self.user_id, 3, 'state_name',
                'description', {'old_content': {}})

feedback_services.create_suggestion(
                self.exploration.id, self.new_user_id, self.exploration.version,
                'a state', 'simple description', {'content': {}})
```

I modified the docstring a bit. PTAL.